flink-cdc-runtime/src/test/java/org/apache/flink/cdc/runtime/operators/transform/PreTransformOperatorTest.java

```diff
@@ -526,4 +526,39 @@ void testMetadataTransform() throws Exception {
                         new StreamRecord<>(
                                 new CreateTableEvent(METADATA_TABLEID, EXPECTED_METADATA_SCHEMA)));
     }
+
+    @Test
+    void testMultiTransform() throws Exception {
+        PreTransformOperator transform =
+                PreTransformOperator.newBuilder()
```

Comment:
Maybe we can also cover cases when projection is `*` or omitted here

Reply:
done~, also above
added some tests